packages/ses/scripts/hermesTransforms.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { parse } from '@babel/parser';
+import babelTraverse from '@babel/traverse';
+import babelGenerate from '@babel/generator';
+import * as t from '@babel/types';
+
+// TODO: do we need to use this instead?
+// import babelGenerator from '@agoric/babel-generator';
+
+// TODO: Stolen from evasive-transform package sources. Is this still necessary?
+const traverse = babelTraverse.default || babelTraverse;
+const generate = babelGenerate.default || babelGenerate;
+
+const decoder = new TextDecoder();
+const encoder = new TextEncoder();
```

Comment:
nb: CI fails with `TextEncoder('utf-8')` on `yarn docs` (`typedoc`)

```console
packages/ses/scripts/hermesTransforms.js:17:33 - error TS2554: Expected 0 arguments, but got 1.

17 const encoder = new TextEncoder('utf-8');
```

Reply:
The web has not done a good job of proofing `TextEncoder` for a future where it could support other codecs, so it’s unlikely to ever honor an argument.
```js
> new TextEncoder('latin-1')
{ encoding: 'utf-8' }
```